src/main/java/dk/kb/present/View.java

```diff
@@ -85,30 +88,55 @@ public MediaType getMime() {
     }
 
     @Override
-    public String apply(String recordID, String content, String child) {
-        final Map<String, String> metadata = new HashMap<>();
-        metadata.put("recordID", recordID);
-        metadata.put("origin", origin);
+    public String apply(DsRecordDto record) {
+        final Map<String, String> metadata = createBasicMetadataMap(record);
 
-        if (child != null ) {
-            metadata.put("childRecord", child);
-        }
+        String content = record.getData();
+
+        switch (strategy) {
+            case "manifestation":
+                updateMetadataMapWithPreservicaManifestation(record, metadata);
+            case "none":
 
+        }
```

Comment:
When using a `switch`, please add a `default` that throws an `UnsupportedOperationException` with the given value.

Reply:
Nice error message :heart: